apps/ui/src/components/SpaceTreasury.vue

```diff
@@ -130,6 +130,7 @@ onMounted(() => {
     return router.replace({
       name: 'space-treasury',
       params: {
+        index: route.params.index || 1,
```

Comment:
That solves the issue, but I'm thinking maybe we should watch it instead of `onMounted`, changing it via URL now doesn't do anything unless you reload.

Reply:
Updated it